latch_cli/services/local_dev.py

```diff
@@ -233,22 +266,32 @@ def file_filter(filename: str) -> bool:
                                 continue
 
                             if cmd in QUIT_COMMANDS:
-                                await aioconsole.aprint(
-                                    "Exiting local development session"
-                                )
-                                await ws.close()
-                                return
-
-                            if cmd.startswith("run"):
-                                await aioconsole.aprint(
-                                    "Syncing your local changes... "
-                                )
+                                raise EOFError
+
+                            if not COMMAND_EXPRESSION.match(cmd):
+                                await aioconsole.aprint("Invalid command")
```

Comment:
parsing out useful errors from a failed match would be instructive here (eg. run-scripts vs run-script often trips me up)

Reply:
also a good call